zcash_client_backend/CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this library adheres to Rust's notion of
   - `impl Default for wallet::input_selection::GreedyInputSelector`
 - `zcash_client_backend::fees::SplitPolicy`
 - `zcash_client_backend::fees::zip317::MultiOutputChangeStrategy`
+- `zcash_client_backend::fees::standard::MultiOutputChangeStrategy`
```

Comment:
Does this exist? I don't see it being introduced in the commit that adds this line (2815d27594c27aef5c8c4c8294ea79335516ba62).

EDIT: It gets added in a later commit.